src/Symfony/Component/Validator/Tests/Mapping/Factory/LazyLoadingMetadataFactoryTest.php

```diff
@@ -20,15 +20,17 @@ class LazyLoadingMetadataFactoryTest extends \PHPUnit_Framework_TestCase
 {
     const CLASSNAME = 'Symfony\Component\Validator\Tests\Fixtures\Entity';
     const PARENTCLASS = 'Symfony\Component\Validator\Tests\Fixtures\EntityParent';
-    const INTERFACECLASS = 'Symfony\Component\Validator\Tests\Fixtures\EntityInterface';
+    const INTERFACEACLASS = 'Symfony\Component\Validator\Tests\Fixtures\EntityInterfaceA';
```

Comment:
Could you please add some underscores to constants names to ease readability, I know you did it for consistency because there weren't any but since you add some more, and longer ones with single-lettered meanings... I think maybe we could start using them :)